docs/docs/config/kscan.md

```diff
@@ -0,0 +1,309 @@
+---
+title: Keyboard Scan Configuration
+sidebar_label: Keyboard Scan
+---
+
+See [Configuration Overview](index.md) for instructions on how to change these settings.
+
+## Common
+
+### Kconfig
+
+Definition files:
+
+- [zmk/app/Kconfig](https://github.com/zmkfirmware/zmk/blob/main/app/Kconfig)
+- [zmk/app/drivers/kscan/Kconfig](https://github.com/zmkfirmware/zmk/blob/main/app/drivers/kscan/Kconfig)
+
+| Config                                 | Type | Description                                          | Default |
+| -------------------------------------- | ---- | ---------------------------------------------------- | ------- |
+| `CONFIG_ZMK_KSCAN_EVENT_QUEUE_SIZE`    | int  | Size of the event queue for kscan events             | 4       |
+| `CONFIG_ZMK_KSCAN_INIT_PRIORITY`       | int  | Keyboard scan device driver initialization priority  | 40      |
+| `CONFIG_ZMK_KSCAN_DEBOUNCE_PRESS_MS`   | int  | Global debounce time for key press in milliseconds   | -1      |
+| `CONFIG_ZMK_KSCAN_DEBOUNCE_RELEASE_MS` | int  | Global debounce time for key release in milliseconds | -1      |
+
+If the debounce press/release values are set to any value other than `-1`, they override the `debounce-press-ms` and `debounce-release-ms` devicetree properties for all keyboard scan drivers which support them.
```

Comment:
If I understand correctly these settings only apply to the matrix driver at the moment, we should note that if that is the case.

Reply:
Added notes about this.